src/redux/history/methods.ts

```diff
@@ -0,0 +1,212 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { store } from '../index'
+import {
+  HistoryActionType,
+  HistoryEntry,
+  HistoryEntryOrigin,
+  HistoryExportJson,
+  RemoveEntryAction,
+  SetEntriesAction,
+  UpdateEntryAction,
+  V1HistoryEntry
+} from './types'
+import { download } from '../../components/common/download/download'
+import { DateTime } from 'luxon'
+import {
+  deleteHistory,
+  deleteHistoryEntry,
+  getHistory,
+  postHistory,
+  updateHistoryEntryPinStatus
+} from '../../api/history'
+import {
+  historyEntryDtoToHistoryEntry,
+  historyEntryToHistoryEntryPutDto,
+  historyEntryToHistoryEntryUpdateDto
+} from '../../api/history/dto-methods'
+
+type ErrorHandler = (message: string) => void
+
+export const setHistoryEntries = (entries: HistoryEntry[], onError?: ErrorHandler): void => {
+  store.dispatch({
+    type: HistoryActionType.SET_ENTRIES,
+    entries
+  } as SetEntriesAction)
+  storeLocalHistory()
+  storeRemoteHistory(onError)
```

Comment:
I though you wanted to remove this call?

Reply:
Yes and no. I wanted to remove it for the deleteHistory, deleteHistoryEntry, updateHistoryEntry etc. and did so, but not here.

But I just see, that I could rearrange something here too to remove another unnecessary backend call.

Reply:
Changed something